kothic/style/mapcss.js

```diff
@@ -301,6 +301,21 @@ var MapCSS = {
 		return lst;
 	},
 
+	e_length: function(str) {
+		if (Object.prototype.toString.call(str) !== '[object String]')
+			return 0;
+
+		return str.length;
+	},
+
+	e_substring: function(str, index) {
+		return str.substr(index);
+	},
+
+	e_replace: function(s, target, replace) {
+		return s.split(target).join(replace);
```

Comment:
Why don't you use https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/String/replace instead?